### PR TITLE
request-response scenario's openApi definition endpoint

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1019,15 +1019,20 @@ lazy val liteEngineKafkaIntegrationTest: Project = (project in lite("kafka/integ
     name := "nussknacker-lite-kafka-integration-test",
     IntegrationTest / Keys.test := (IntegrationTest / Keys.test).dependsOn(
       liteEngineKafkaRuntime / Universal / stage,
-      liteEngineKafkaRuntime / Docker / publishLocal
+      liteEngineKafkaRuntime / Docker / publishLocal,
+      liteRequestResponseComponents / Compile / assembly,
```
Comment (dswiecki) [inline build.sbt:1023]: is this really needed ?

File: engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuReqRespRuntimeDockerTest.scala [hunk 2]
```diff
@@ -28,4 +28,68 @@ class NuReqRespRuntimeDockerTest extends FunSuite with ForAllTestContainer with
     request.body(jsonPingMessage("dockerFoo")).send().body shouldBe Right(jsonPongMessage("dockerFoo"))
   }
 
+  test("should get scenario definition via http") {
```
Comment (dswiecki) [inline engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuReqRespRuntimeDockerTest.scala:31]: why this test class is inside .kafka package, lets change it as in main sources

File: engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/requestresponse/ScenarioRouteSpec.scala [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+import java.io.File
+import scala.concurrent.Future
+
+
+class ScenarioRouteSpec extends FlatSpec with ScalatestRouteTest with Matchers {
+
+  import spel.Implicits._
+
+  private val inputSchema = "{\"properties\": {\"city\": {\"type\": \"string\", \"default\": \"Warsaw\"}}}"
```
Comment (dswiecki) [inline engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/requestresponse/ScenarioRouteSpec.scala:28]: lets change for multiline string

File: engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuReqRespRuntimeDockerTest.scala [hunk 4]
```diff
@@ -28,4 +28,68 @@ class NuReqRespRuntimeDockerTest extends FunSuite with ForAllTestContainer with
...
+    val definitionReq = basicRequest.get(uri"http://${runtimeContainer.host}".port(mappedRuntimeApiPort).path("scenario", pingPongScenario.id, "definition"))
+
+    val definition = definitionReq.send().body.rightValue
+
+    definition shouldBe s"""{
```
Comment (dswiecki) [inline engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuReqRespRuntimeDockerTest.scala:36]: make the same as in expectedOApiDef ?

File: engine/lite/request-response/components-api/src/main/scala/pl/touk/nussknacker/engine/requestresponse/api/openapi/RequestResponseOpenApiSettings.scala [hunk 5]
```diff
@@ -4,6 +4,6 @@ object RequestResponseOpenApiSettings {
 
   val OutputSchemaProperty = "outputSchema"
   val InputSchemaProperty = "inputSchema"
-  val OPEN_API_VERSION = "3.0.0"
+  val OPEN_API_VERSION = "3.0.3"
```
Comment (dswiecki) [inline engine/lite/request-response/components-api/src/main/scala/pl/touk/nussknacker/engine/requestresponse/api/openapi/RequestResponseOpenApiSettings.scala:7]: I did'n know we have such a version here, couldn't it be 3.1.0 ?

File: build.sbt (hunk 1 above)
Comment (gadomsky) [reply build.sbt]: no

**Review by dswiecki — APPROVED** [review]

**Pull request merged**